src/control/cmd/daos_server/support.go

```diff
@@ -14,6 +14,7 @@ import (
 
 	"github.com/daos-stack/daos/src/control/common/cmdutil"
 	"github.com/daos-stack/daos/src/control/lib/support"
+	"github.com/pkg/errors"
```

Comment:
nit: The convention for imports is:

stdlib

vendored

internal

Reply:
done